components/GameCard.js

```diff
@@ -0,0 +1,40 @@
+import Image from "next/image";
+import styled from "styled-components";
+
+export default function GameCard({ game }) {
+  return (
+    <StyledCard>
+      <StyledTitle>{game.name.value}</StyledTitle>
+      <StyledRankTitle>{game.rank}</StyledRankTitle>
+      <Image
+        src={game.thumbnail.value}
+        alt={game.name.value}
+        width={100}
+        height={100}
+      ></Image>
+      <p>{game.yearpublished.value}</p>
+    </StyledCard>
+  );
+}
+
+const StyledRankTitle = styled.p`
+  color: #ff8200;
+`;
+const StyledCard = styled.div`
+  width: 80%;
```

Comment:
instead of using this percentage wise padding, please give the container element of the Cards a padding or a max-width value.

Reply:
const StyledCard = styled.div`
  max-width: 80%;
  }
`;